docs/operational_guide/kernel_configuration.md

```diff
@@ -31,12 +31,29 @@ To set this value permanently, update the `vm.swappiness` setting in `/etc/sysct
 ## rlimits
 M3DB also can use a high number of files and we suggest setting a high max open number of files due to per partition fileset volumes.
 
-On Linux you can set a high limit for maximum number of open files in `/etc/security/limits.conf`:
+On Linux you can set a high limit for the maximum number of open files for a specific user in `/etc/security/limits.conf`:
 ```
-your_m3db_user        hard nofile 500000
-your_m3db_user        soft nofile 500000
+your_m3db_user        hard nofile 3000000
+your_m3db_user        soft nofile 3000000
 ```
 
+In addition, you may need to override the system and process-level limits set by the kernel with the following commands:
+
+Set the kernel-wide limit to 3 million.
```

Comment:
Can we say that most kernels come w/ a higher value for this and only set this if your val is lower